app/transcription.py

```
@@ -453,36 +515,326 @@ def remove_transcription_source_JSON(self, json_file):
+        # (without pipeline_state) BEFORE start() is called, so the "latest" file
+        # is often a fresh blank one — not the persisted state from a previous run.
+        saved = None
+        for filepath in reversed(existing_files):
+            try:
+                with open(filepath) as fh:
+                    data = json.load(fh)
+                if "pipeline_state" in data:
+                    saved = data["pipeline_state"]
+                    break
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)